src/holdem/parse.rs

```diff
@@ -666,11 +692,27 @@ mod test {
         assert_eq!(6 * 2, count);
     }
 
+    #[test]
+    fn test_parse_sequential() {
+        assert_eq!(
+            RangeParser::parse_one(&String::from("A9s-A5s"))
+                .unwrap()
+                .len(),
+            20
+        );
```

Comment:
Please add a test showing that each hand satisfies the invariants.

- Every hand has an Ace in it.
- The other card is between 9 and 5
- Both cards are the same suit